.github/actions/build-docsite/script.sh

```diff
@@ -9,7 +9,6 @@ pip install awscli
 
 make
 sleep 1
-rm _book/.travis.yml
```

Comment:
(I take it this is being removed because there's no longer a `.travis.yml` in the project?)

Reply:
Yeah travis did a migration that broke this project, and we determined it was easier to just move it to github actions